README.md

```diff
@@ -1,9 +1,84 @@
 
-Currently, dccmonitor is a collection of functions to gather data of interest from files within a Synapse file view. The goal is to add a shiny app dashboard for viewing the information.
+**Note:** The application takes time to fully populate the validation information and currently does not have a progress indicator. The application will initialize the study boxes after checking for team membership and gathering the studies represented in the `consortium_fileview` (see Customization for details on this file view). After this, the application will populate the study boxes after the validation checks have run for all represented studies.
 
-## Installation
+## Requirements
 
-`devtools::install_github("Sage-Bionetworks/dccmonitor")`
\ No newline at end of file
+dccmonitor uses the reticulate package with the [Synapse Python Client](https://github.com/Sage-Bionetworks/synapsePythonClient). See the [reticulate documentation](https://rstudio.github.io/reticulate/#python-version) for more information on setting up reticulate to work with your local Python environment. Additionally, see the [Synapse Python Client](https://github.com/Sage-Bionetworks/synapsePythonClient) for installation instructions.
+
+Using the dccmonitor Shiny application requires that the user have a Synapse account, and have permission to access necessary Synapse files. These files include the specific Synapse folder of interest for monitoring (see [Customization](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#customization) for details), along with access to all dccvalidator Synapse dependencies (ex: template and annotation files).
+
+## Installation and Use
+
+dccmonitor can be installed and used in two ways: package installation, and cloning the repository.
+
+### Package Installation
+
+dccmonitor can be installed via devtools:
+
+``` R
+devtools::install_github("Sage-Bionetworks/dccmonitor")
+```
+
+To run the Shiny application, there needs to be a config.yml file in the working directory with the required options (see [Customization](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#customization)). After creating the config.yml file, the application can be run with:
+
+``` R
+library(dccmonitor)
+Sys.setenv(R_CONFIG_ACTIVE = "default") # Replace "default" with the configuration name if not using default.
+run_app()
+```
+
+### Cloning Repository
+
+[Clone the repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) to a local directory. Customize the config.yml file, as needed (see [Customization](https://github.com/Sage-Bionetworks/dccmonitor/tree/update-readme#customization)).
+
+If your working directory is the application directory, you can change the app.R file to have your configuration name, if not using the default configuration. Then the following will start the app:
```

Comment:
I suggest providing an example of how to change the configuration from default.

Reply:
I'm not sure what to put as an example for that since it's changing a single word in a file. The app.R file has a comment in it that says where to change the default. Do you have an example so I can better understand what you are looking for?

Reply:
I took a look at the app.R file and I see your comment which is clear enough.

Reply:
Great!